EMML/1001-2000/EMML1832.xml

```diff
@@ -31,12 +31,113 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
          <sourceDesc>
             <msDesc xml:id="ms">
                <msIdentifier>
-                  <repository ref="INS0004HMML"/>
-                  <collection>Monastery of Ḥayq Esṭifānos, Ambāssal</collection>
-                  <idno>MS EMML no. 1832</idno>
+                  <repository ref="INS0327DHE"/>
+                  <collection>Monastery of Ḥayq ʾƎsṭifānos, Ambāssal</collection>
+                  <idno>EMML 1832</idno>
```

Comment:
OK but add 
`<altIdentifier>
                     <repository ref="INS0004HMML"/>
                     <collection>Ethiopian Manuscript Microfilm Library</collection>
                     <idno>EMML 1832</idno>
                  </altIdentifier>`

Reply:
Yes. I guess, it must be inside `<msIdentifier>`